frontend/src/metabase/search/components/InfoText/InfoText.tsx

```diff
@@ -104,30 +128,37 @@ export const InfoTextTablePath = ({ result }: InfoTextProps) => {
 
   return (
     <>
-      {showDatabaseLink && <DatabaseLink database={database} />}
+      {showDatabaseLink && (
+        <DatabaseLink isEnabled={isEnabled} database={database} />
+      )}
       {showTableLink && (
         <>
           {LinkSeparator}
-          <TableLink result={result} />
+          <TableLink isEnabled={isEnabled} result={result} />
```

Comment:
nit: If you're bothered by too much props passing, we could inline `DatabaseLink` and `TableLink` components here, as they're small and only used here.

Reply:
It'll be fine for now